client/src/app/site/pages/meetings/modules/projector/modules/slides/components/motions/modules/motion-slide/components/motion-slide/motion-slide.component.ts

```diff
@@ -154,6 +154,15 @@ export class MotionSlideComponent
 
     private _showText = false;
 
+    protected get hideMetadataBackground(): boolean | undefined {
+        if (this._hideMetadataBackground === undefined) {
+            return false;
+        }
+        return this._hideMetadataBackground;
+    }
+
+    private _hideMetadataBackground = false;
+
```

Comment:
Using an accessor for this is a but unnecessary

Reply:
removed the accessor